internal/tsc_wrapped/tsconfig.json

```diff
@@ -10,8 +10,9 @@
       "es5",
       "es2015.core",
       "es2015.collection",
+      "es2015.promise",
+      // TODO: Remove "es2015.iterable" by replacing all instances of "for .. of".
```

Comment:
This needs some discussion. I don't see how we would practically change user behavior.

I don't think it's our job to tell all users how to write ES2015 that performs well on all platforms. They'd have this problem even without using any Bazel stuff.

Presumably the browsers get faster at running `for..of` loops in the future, then users change back? Seems like churn.

Reply:
This is mainly for the performance of tsc_wrapped, in the vast majority of use cases the you'll never see a practical change in execution time, unless it's something like a compiler wrapper.

Not to mention you'll gain much more perf in all the other areas.

For .. of will never be faster the for.